src/main/kotlin/org/malachite/estella/config/WebSecurityConfig.kt

```diff
@@ -28,6 +38,7 @@ class WebSecurityConfig(val jwtFilter: JwtSecurityFilter, val userDetailsService
         http!!
                 .cors().and()
                 .csrf().disable()
+                .headers().frameOptions().disable().and()
```

Comment:
Co to robi?

Reply:
podejrzewam, że pozwala na niestandardowe nagłówki

Reply:
To jest header, który mówi czy strona powinna wyświetlać inne strony (w tym swoje podstrony) w elementach html. Potrzebne, żeby konsola h2 się poprawnie wyświetlała, myślę że to jeden z wielu nagłówków o których istnieniu warto wiedzieć, bo może uderzyć w twarz xd